onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4223,6 +4223,30 @@ def aten_index_put(
     See implementation of `torch.onnx.symbolic_opset11.index_put
     <https://github.com/pytorch/pytorch/blob/main/torch/onnx/symbolic_opset11.py#L212>`_.
     """
+    if (
+        len(indices) > 1
+        and any(
+            isinstance(index, torch.onnx._internal.exporter._tensors.SymbolicTensor)  # pylint: disable=protected-access
+            for index in indices
+        )
+        and len(values.shape) == 1
+    ):
+        return _aten_index_put_dynamic(self, indices, values, accumulate=accumulate)
+
+    n_none = [i for i, ind in enumerate(indices) if ind is not None]
```

Comment:
I guess `n_none` is short for `not_none` ... may be better to call it `not_none`